### PR TITLE
fix(theme): fix export maps

### PR DESCRIPTION
**Pull request by gauriGupta99**
## Previous Behavior
The export maps for the CommonJS paths are incorrectly configured, causing exceptions when used.

## New Behavior
Fix exports maps and point them to correct location.

## Related Issue(s)

- Fixes #
- Fixed common-js export maps to point to correct path

### CODE REVIEW

File: change/@fluentui-theme-f00b418d-ad44-4121-8926-2c1c1949d550.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-theme-f00b418d-ad44-4121-8926-2c1c1949d550.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31484&commitId=dd5a45d4c36814f64ba75d6c2c3b3f7a87b2e22e&env=prod&clientType=FLUENTUI)


  <details><summary><b>Pivot - Overflow</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

**Review by Hotell — APPROVED** [review]
ty!

**Review by spmonahan — APPROVED** [review]

**Pull request merged**